package.json

```diff
@@ -35,11 +37,13 @@
     "@microsoft/api-extractor": "^7.19.4",
     "@typescript-eslint/eslint-plugin": "^4.28.2",
     "@typescript-eslint/parser": "^4.28.2",
-    "eslint": "^7.30.0",
+    "eslint": "^7.32.0",
```

Comment:
Out of curiosity, is there a reason why the version went up?

Reply:
I ran `npm install prettier eslint husky --save-dev` to install packages and did not notice eslint has already installed. But it should not matter in terms of behavior.